### PR TITLE
Update fonts & colors

### PR DESCRIPTION
**Pull request by anatoly-particle**
• | Before | After
--- | --- | ---
📸 | old | new
🤖| before | after

### CODE REVIEW

File: templates/partials/head.hbs [hunk 1]
```diff
@@ -9,7 +9,7 @@
 <link rel="stylesheet" href="https://s3.amazonaws.com/icomoon.io/49057/Dashboard/style.css?9t9e91">
 
 <!-- GOTHAM YO -->
-<link rel="stylesheet" type="text/css" href="//cloud.typography.com/6107112/665726/css/fonts.css" />
+<!-- <link rel="stylesheet" type="text/css" href="//cloud.typography.com/6107112/665726/css/fonts.css" /> -->
```
Comment (monkbroc) [inline templates/partials/head.hbs]: You can remove this completely

File: src/assets/less/fonts.less [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+                 url("https://fonts.particle.io/1c234575-4907-46fb-a87c-4ab80464bb84.ttf") format('truetype');
+}
+
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ Fira Mono (monospaced) ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+@font-face {
+    font-family: 'Fira Mono';
+    font-style : normal;
+    font-weight: 500; /* Medium */
+    src        : url("https://s3.amazonaws.com/fonts.particle.io/FiraMono-Medium.woff2") format('woff2'),
```
Comment (monkbroc) [inline src/assets/less/fonts.less]: Can you switch all the FiraMono URLs to fonts.particle.io (like https://fonts.particle.io/FiraMono-Medium.woff2) to take advantage of caching provided by Cloudfront?

File: templates/redirector.html.hbs [hunk 3]
```diff
@@ -11,7 +11,7 @@
     </script>
     <link href="/assets/orange.ico" rel="shortcut icon" type="image/x-icon">
     <link href="/assets/css/style.css" rel="stylesheet">
-    <link href="//cloud.typography.com/6107112/665726/css/fonts.css" rel="stylesheet" type="text/css">
+    <!-- <link href="//cloud.typography.com/6107112/665726/css/fonts.css" rel="stylesheet" type="text/css"> -->
```
Comment (monkbroc) [inline templates/redirector.html.hbs]: Remove completely

**Review by monkbroc — APPROVED** [review]
Looks good

**Comment by monkbroc** [comment]
Can you add the font license snippet from https://github.com/particle-iot/website-prototype/blob/master/gatsby-browser.js#L48 to the scripts partial https://github.com/particle-iot/docs/blob/6ae6549f4184fdfd1d5681f669d4a2f03601d78b/templates/partials/scripts.hbs#L31

**Comment by monkbroc** [comment]
The font size could be increased 15% across the board to improve readability (14px => 16px, etc). The new fonts look a little smaller than the old ones.

**Comment by anatoly-particle** [comment]
Thank you for pointing to the license snippet!

**Review by monkbroc — APPROVED** [review]

**Comment by anatoly-particle** [comment]
@monkbroc I've updated the top post to reflect the changes.

**Pull request merged**